content/NavCoin Core/How to move from NavPi to Raspberry Pi 4 and CM4.en.md

```diff
@@ -0,0 +1,182 @@
+---
+title: How to move from NavPi to Raspberry Pi 4 and CM4 for staking
```

Comment:
I know you started working on this with the purpose of moving from navpi, but it may be beneficial to have a generic tittle for headless setup guide and single out navpi steps in the content.

Reply:
agreed - makes total sense - will change in next revision